src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```diff
@@ -109,6 +109,11 @@ internal async Task InitializeRpcServerAsync()
 
         internal Task InitializeChannelsAsync()
         {
+            if (_environment.IsInProc())
+            {
+                return Task.CompletedTask;
+            }
```

Comment:
once the other comment is complete -- you shouldn't need this and we can leave this class as-is.

Reply:
I like this idea